deployments/daemon-set/nginx-ingress.yaml

```diff
@@ -16,6 +16,14 @@ spec:
        #prometheus.io/port: "9113"
     spec:
       serviceAccountName: nginx-ingress
+      securityContext:
```

Comment:
does it make sense to define the security context at the container level? those only make sense for the nginx container.  although we don't have any other container, what if somebody would want to add some sidecar container to the Ingress Controller pod?

Reply:
Interesting question. Yes, I think you are right, we shouldn't set capabilities for the whole pod but just for the IC container. Will change, thanks for the heads up!

Reply:
Please note that changing this to the container will require `allowPrivilegeEscalation: true` instead of `false` (even with setcap). Otherwise, we get:

```
2019/09/30 13:59:15 [emerg] 17#17: bind() to 0.0.0.0:80 failed (13: Permission denied)
nginx: [emerg] bind() to 0.0.0.0:80 failed (13: Permission denied)
```